pytorch_lightning/plugins/ddp_plugin.py

```diff
@@ -59,15 +60,16 @@ def configure_ddp(self, model, device_ids):
             the model wrapped in LightningDistributedDataParallel
 
         """
-        # if unset, default `find_unused_parameters` `True`
+        # if unset, default `find_unused_parameters` `False`
         self._ddp_kwargs["find_unused_parameters"] = self._ddp_kwargs.get(
-            "find_unused_parameters", True
+            "find_unused_parameters", False
         )
         model = LightningDistributedDataParallel(
             model,
             device_ids=device_ids,
             **self._ddp_kwargs,
         )
+        self.device_ids = device_ids
```

Comment:
not sure about this, setting the device ids in every subclass is not so convenient

Reply:
Done !